project-base/src/Controller/Front/CartController.php

```diff
@@ -110,8 +110,7 @@ public function __construct(
     public function indexAction(Request $request)
     {
         $cart = $this->cartFacade->findCartOfCurrentCustomerUser();
-        $cartItems = $cart === null ? [] : $cart->getItems();
-
+        $cartItems = $cart === null ? [] : array_values($cart->getItems());
```

Comment:
I have been thinking about this. On first look it might appear to someone like not necessary code and might want to delete it in future. I would add method getValues to Cart like:
```
public function getValues()
{
    $this->items->getValues();
}
```

and use that method here.

Reply:
I dissagree. The method you are suggesting is not helping at all. I would rather add comment on line above with an explanation of doing so. Then I will pray for reworking CartItems to contain even prices - having it separated is really hell.